docs/src/main/asciidoc/security-jpa.adoc

```diff
@@ -102,7 +103,7 @@ public class User extends PanacheEntity {
     @Password
     public String pass;
 
-    @ManyToMany
+    @ManyToMany(cascade = CascadeType.ALL)
```

Comment:
@jedla97 Does it have to be added here ? It seems to work without it ?

Reply:
@sberyozkin I look on this. Now I probably understand it more, mainly what this part should say. First I was just switch from roles defined as string (previous example) and use this. This caused the need for this. From my understanding is that this section showing that user roles can be stored as list.

Main point for this is that if we need to do some operation with user (add new role) it need the define cascade (at least I didn't find any way how to not use it). I'll update the PR as this don't need be part of the code, but add note as it can be useful for people reading this guide. If you won't any note about this please let me know.

Reply:
@jedla97 Thanks, yes, please add a note with some advice

Reply:
Updated. Sorry for taking that long I had more important things on my list.

Reply:
Np at all,  thanks @jedla97